common/src/logger/mod.rs

```diff
@@ -139,3 +139,19 @@ cfg_if::cfg_if! {
         pub use loggers::*;
     }
 }
+
+/// Creates a test logger for the current function.
```

Comment:
Thinking this should probably be inside of, [loggers/mod.rs](https://github.com/mobilecoinfoundation/mobilecoin/blob/213c8f8bde6059bd4a19e766be99208b35e52493/common/src/logger/loggers/mod.rs#L1)

It's a macro so think it only matters if used directly, but the `create_test_logger` is hidden behind [`if #[cfg(all(feature = "log", feature = "loggers"))] {`](https://github.com/mobilecoinfoundation/mobilecoin/blob/213c8f8bde6059bd4a19e766be99208b35e52493/common/src/logger/mod.rs#L137) so it seems like this macro should be hidden behind the same features